src/components/BuildBaronAndAnnotations/AnnotationTickets.tsx

```diff
@@ -64,7 +64,7 @@ export const AnnotationTickets: React.FC<Props> = ({
   ) : null;
 };
 
-const StyledButton = styled(PlusButton)`
+export const StyledButton = styled(PlusButton)`
```

Comment:
Is this used anywhere else?

Reply:
Ah I was using it in AnnotationNote for a bit but it didn't do what I wanted so I changed it, I'll remove the export.